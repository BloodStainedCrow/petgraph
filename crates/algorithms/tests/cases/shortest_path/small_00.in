version https://git-lfs.github.com/spec/v1
oid sha256:57bcca3d9e99a79b2a3e2c560a944dac6f0647461ab51384b2e853179128396d
size 877
