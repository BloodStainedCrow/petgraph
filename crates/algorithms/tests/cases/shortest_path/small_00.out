version https://git-lfs.github.com/spec/v1
oid sha256:1cda3f5e6b776c0087aec2c2482c35162a0d34ee43b5601d192d8c1710d874da
size 35
